core/src/main/java/com/novoda/noplayer/NoPlayerView.java

```diff
@@ -29,15 +30,16 @@ public NoPlayerView(Context context, AttributeSet attrs) {
     public NoPlayerView(Context context, AttributeSet attrs, int defStyleAttr) {
         super(context, attrs, defStyleAttr);
         surfaceHolderProvider = new PlayerViewSurfaceHolder();
-        aspectRatioChangeListener = new AspectRatioChangeListener(this);
+        aspectRatioChangeListener = new AspectRatioChangeForwarder(this);
     }
 
     @Override
     protected void onFinishInflate() {
         super.onFinishInflate();
         View.inflate(getContext(), R.layout.noplayer_view, this);
-        videoFrame = Views.findById(this, R.id.video_frame);
+        videoFrame = Views.findById(this, R.id.exo_content_frame);
```

Comment:
We could, and I am not sure how feasible this is, just use a plain `SurfaceView` for both `MediaPlayer` and `ExoPlayer`. This would likely be a lot of work for `ExoPlayer2` because the `SimpleExoPlayerView` does a lot of setup for rendering internally.